_episodes_rmd/02-r-basics.Rmd

```diff
@@ -8,18 +8,18 @@ questions:
 - "What are the most common objects in R?"
 objectives:
 - "Be able to create the most common R objects including vectors"
-- "Understand that vectors have modes, which correspond to the type of data they
+- "Understand that vectors have modes, which correspond to the types of data they
```

Comment:
A single vector can only contain a single type of data, so I'll keep the singular "type" here.